reopt_api/staging_settings.py

```diff
@@ -112,7 +112,7 @@
     'default':{
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
         'HOST': staging_database_host,
-        'NAME': staging_database_name,
+        'NAME': 'reopt_chp_staging_external',
```

Comment:
@t-kwasnik how should we handle migrating the data from reopt_chp_staging_external to the production database?

Reply:
@NLaws I can pull up some scripts I sent to Nick M a while back. After we deploy I can kick off scripts to copy the data over from my local machine or an NREL server

Reply:
@NLaws @t-kwasnik Sorry, this reopt_chp_staging_external was only adopted for the recent chpbeta_chiller_flat branch. The external testers (AMO TAPs) have been using the reopt_staging database (same as develop uses), so that's the one that would need to be migrated. This got into chpbeta when I merged chpbeta_chiller_flat into chpbeta, but chpbeta has not since been deployed to staging.